bounties_api/std_bounties/views.py

```diff
@@ -142,6 +142,16 @@ class CategoryViewSet(viewsets.ReadOnlyModelViewSet):
     search_fields = ('normalized_name',)
 
 
+class LanguageViewSet(viewsets.ReadOnlyModelViewSet):
+    serializer_class = LanguageSerializer
+    queryset = Language.objects.order_by('name')
+    filter_class = LanguageFilter
+    filter_backends = (OrderingFilter, SearchFilter, DjangoFilterBackend,)
+    ordering_fields = ('name',)
+    ordering = ('name',)
```

Comment:
Interesting - for now we could probably get rid of the ordering_fields or ordering since it's unlikely the endpoint will want to sort it non-alphabetically or in any other format.

Reply:
But good you got to play with this.

Reply:
👍